<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainInterface</class>
 <widget class="QMainWindow" name="MainInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph Visualization</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/Icon/analytics.png</normaloff>:/Icon/analytics.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="menu" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#menu{
background-color:#FAFAFA;
}
#BFile{
background-color:#FAFAFA;
border:none;
}
#BFile:hover{
background-color:#E0E0E0;
    color: white;
}
#BHome{
background-color:#FAFAFA;
border:none;
}
#BHome:hover{
background-color: #E0E0E0;
    color: white;
}
#BInsert{
background-color:#FAFAFA;
border:none;
}
#BInsert:hover{
background-color:#E0E0E0;
    color: white;
}
</string>
    </property>
    <widget class="QPushButton" name="BFile">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BHome">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BInsert">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Insert</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="Table">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>151</y>
      <width>781</width>
      <height>301</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>13</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>14</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>15</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>16</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>17</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>18</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>19</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>20</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>21</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>22</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>23</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>24</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>25</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>26</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>27</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>28</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>29</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>30</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>31</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>32</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>33</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>34</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>35</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>36</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>37</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>38</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>39</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>40</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>41</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>42</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>43</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>44</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>45</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>46</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>47</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>48</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>49</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>50</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>51</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>52</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>53</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>54</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>55</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>56</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>57</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>58</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>59</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>60</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>61</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>62</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>63</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>64</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>65</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>66</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>67</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>68</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>69</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>70</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>71</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>72</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>73</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>74</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>75</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>76</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>77</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>78</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>79</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>80</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>81</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>82</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>83</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>84</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>85</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>86</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>87</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>88</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>89</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>90</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>91</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>92</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>93</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>95</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>94</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>96</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>97</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>98</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>100</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>A</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>B</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>C</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>E</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>F</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>G</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>H</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>I</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>J</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>K</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>L</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>M</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>N</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>O</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>P</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Q</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>R</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>S</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>T</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>U</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>V</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>W</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>X</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Y</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Z</string>
     </property>
    </column>
   </widget>
   <widget class="MenuALL" name="Menu" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>30</y>
      <width>801</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/Icon/cut.png</normaloff>:/Icon/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/Icon/copy.png</normaloff>:/Icon/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/Icon/paste.png</normaloff>:/Icon/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MenuALL</class>
   <extends>QWidget</extends>
   <header>menuall.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
